source/distributed/tria_base.cc

```diff
@@ -1518,8 +1526,8 @@ namespace parallel
           const MPI_Offset my_global_file_position =
             static_cast<MPI_Offset>(global_first_cell) * sizeof(unsigned int);
 
-          // It is very unlikely that a single process has more than 2 billion
-          // cells, but we might as well check.
+          // It is very unlikely that a single process has more than 2
+          // billion cells, but we might as well check.
```

Comment:
If you're going to push "billon" onto the next line, then maybe it makes sense to move the number "2" as well.